uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/grids/impl/BaseGridRendererTest.java

```diff
@@ -113,6 +176,116 @@ public void checkSelectedCellsNotClippedByHeader() {
                                    2);
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void checkRenderHeader() {
+        final BaseGridRendererHelper.RenderingInformation ri = makeRenderingInformation(model,
+                                                                                        Arrays.asList(ROW_HEIGHT, ROW_HEIGHT * 2, ROW_HEIGHT * 3));
```

Comment:
Shouldn't we use `HEADER_HEIGHT`?

Reply:
@jomarko No, since the ```makeRenderingInformation(..)``` expects a ```List``` of "row offsets (y-offsets from the Grid's y-coordinate)" i.e. rows can be of variable height so row 0 may have an offset of ```ROW_HEIGHT```, row 1 an offset of ```ROW_HEIGHT * 2``` and so forth (assuming all rows have the same height). This collection is calculated once to help with rendering - removing the need to calculate a row's "offset" each time it is needed.

Reply:
@jomarko I have updated some tests in my latest commit as row 0 should have an offset of 0, row 1 ```ROW_HEIGHT```, row 2 ```ROW_HEIGHT * 2``` etc.